packages/ui-text-area/src/TextArea/index.js

```diff
@@ -248,7 +251,7 @@ class TextArea extends Component {
       this._container.style.minHeight = height
     }
 
-    this._height = height
+    this._height = height // string type, e.g. '55px'
```

Comment:
This info should be in the proptypes section imho

Reply:
fixed